binder/views.py

```diff
@@ -2985,7 +2985,7 @@ def stats_view(self, request):
 			stats = stats.split(',')
 
 		return JsonResponse({
-			stat: self._get_stat(request, queryset, stat, annotations, include_annotations)
+			stat: self._get_stat(request, queryset, stat, annotations.copy(), include_annotations)
```

Comment:
Maybe add a comment about why we copy instead of just passing the annotations directly